frontend/src/components/LandingPage/Features/Features.js

```diff
@@ -0,0 +1,81 @@
+/* 
+    This component will have a title and a description of the features 
+    Features include: 
+        - snap a picture or manually enter ingredients
+        - getting recipes with ingredients you have
+        - seeing nutritional information
+        - and getting videos to go along with your recipe
+*/
+
+import React from 'react';
+import { Card, CardImg, CardTitle, CardText, CardDeck, CardSubtitle, CardBody, Jumbotron } from 'reactstrap';
+import './Features.css'
+const camera = '/coolcamera.jpeg';
+const nutritional = '/nutritional.jpg';
+const recipes = '/goodrecipe.jpeg';
+const videos = '/videos.jpg';
+
+// including id="pictures" makes the picture format stretch when on split screen but even when on full screen
+// removing id="pictures" will make nutrition card slightly longer
+// constant for the card used throughout this file
+const FeaturesCard = (props) => {
```

Comment:
Can we make this a functional component instead of a const.

Reply:
What do you think about putting the FeaturesCard in a separate file and importing it here instead. That way we can have one component per file to minimize confusion.

Reply:
Absolutely! I created the file for the FeaturesCard function and exported it to the Features function just now